bricks/test_optimizer/__brick__/test/.test_optimizer.dart

```diff
@@ -37,7 +37,7 @@ class _TestOptimizationAwareGoldenFileComparator extends LocalFileComparator {
 
   @override
   Uri getTestUri(Uri key, int? version) {
-    final keyString = key.path;
+    final keyString = key.toFilePath();
```

Comment:
Can we add a test for this?

Reply:
What do you think the best way to test this would be? I haven't been able to get it working with an e2e test because using golden tests depends on Flutter. How is this comparator being tested right now?

Reply:
Can we add e2e tests to the CLI to prevent this regression from happening again? Thanks!